input/templates/my-account-edit-address.hbs

```diff
@@ -22,19 +22,19 @@
               </div>
               <div id="my-account-desktop-content" class="col-sm-9">
                 <div class="address-book-content edit-address-book-content">
-                  <form id="form-edit-my-address-book" action="{{@root.meta._links.editMyAddressBook.href}}" method="POST">
+                  <form id="form-edit-my-address-book" action="{{@root.meta._links.myAddressBookEditAddressSubmit.href}}" method="POST">
```

Comment:
The link that goes to the edit form and the one that submits the form should be different, so usual we use the suffix "Submit" to reference to the one that submits. I have also renamed it a bit because later with the "add address" link it sounds a bit weird "addMyAddressBook", while "myAddressBookAddAddress" sounds better :)